export/cmake/CMakeLists.txt.export

```diff
@@ -333,7 +375,7 @@ if (ENABLE_FORTRAN)
 
     # tests
     add_executable(fortran_example EXCLUDE_FROM_ALL fortran/fortran_example.F90)
-    target_link_libraries(fortran_example libint_f int2)
+    target_link_libraries(fortran_example libint_f int2_static)
```

Comment:
presumably this need `if (TARGET int2_static)` (perhaps this will resolve https://travis-ci.org/evaleev/libint/jobs/566892223#L7977 )? It makes sense to define a variable somewhere since this `if` appears in multiple places

Reply:
This problem is addressed in the the latest commit.